content/guides.md

```diff
@@ -26,14 +26,14 @@ To see ports, keyboard layouts, function keys, and product quickstart guides, fi
 | Laptops | Desktops | Accessories |
 |:-------:|:--------:|:-----------:|
 | Adder WS [(addw1)](https://tech-docs.system76.com/models/addw1/README.html) | Meerkat [(meer5)](https://tech-docs.system76.com/models/meer5/README.html) | Launch [(launch_1)](https://tech-docs.system76.com/models/launch_1/README.html) |
-| Adder WS [(addw2)](https://tech-docs.system76.com/models/addw2/README.html) | Meerkat [(meer6)](https://tech-docs.system76.com/models/meer6/README.html) |
-| Bonobo WS [(bonw14)](https://tech-docs.system76.com/models/bonw14/README.html) | Thelio Mira R1 [(thelio-mira-r1)](https://tech-docs.system76.com/models/thelio-mira-r1.0/README.html) |
-| Darter Pro [(darp6)](https://tech-docs.system76.com/models/darp6/README.html) | Thelio Mira B1 [(thelio-mira-b1)](https://tech-docs.system76.com/models/thelio-mira-b1.0/README.html)
-| Darter Pro [(darp7)](https://tech-docs.system76.com/models/darp7/README.html) | Thelio Major [(thelio-major)](https://tech-docs.system76.com/models/thelio-major-intel-and-amd/README.html) |
-| Darter Pro [(darp8)](https://tech-docs.system76.com/models/darp8/README.html) | Thelio Mega R1 [(thelio-mega-r1)](https://tech-docs.system76.com/models/thelio-mega-r1.0/README.html) |
-| Galago Pro [(galp4)](https://tech-docs.system76.com/models/galp4/README.html) | Thelio Massive B1 [(thelio-massive-b1)](https://tech-docs.system76.com/models/thelio-massive-b1.2/README.html) |
-| Galago Pro [(galp5)](https://tech-docs.system76.com/models/galp5/README.html) |
-| Galago Pro [(galp6)](https://tech-docs.system76.com/models/galp6/README.html) |
+| Adder WS [(addw2)](https://tech-docs.system76.com/models/addw2/README.html) | Meerkat [(meer6)](https://tech-docs.system76.com/models/meer6/README.html)  | Luanch [(launch_2)](https://tech-docs.system76.com/models/launch_2/README.html) |
+| Bonobo WS [(bonw14)](https://tech-docs.system76.com/models/bonw14/README.html) | Thelio B4 [(thelio-b4)](https://tech-docs.system76.com/models/thelio-b4/README.html) | Launch Lite [(launch_lite_1)](https://tech-docs.system76.com/models/launch_lite_1/README.html) |
+| Darter Pro [(darp6)](https://tech-docs.system76.com/models/darp6/README.html) | Thelio Major [(thelio-major)](https://tech-docs.system76.com/models/thelio-major-intel-and-amd/README.html) |  Launch Heavy [(launch_heavy_1)](https://tech-docs.system76.com/models/launch_heavy_1/README.html) |
```

Comment:
The link for `Thelio Major [(thelio-major)]` should be changed to `Thelio Major [(thelio-major-r3)]`